### DISCUSSION TITLE
Wrong metadata detection, filename or title?

### DISCUSSION THREAD

**Discussion by schoerg** [jellyfin/jellyfin#9911] [General]
```
Version: 10.8.10
Operating System: Windows
Architecture: X64

Activated metadata provider: 
TheMovieDb
The Open Movie Database
```

I have activated "Prefer embedded titles over filenames" in my library. I am not sure what this means, titles from the MKV title tag? Or filename? [This section](https://jellyfin.org/docs/general/server/media/movies
) does not mention it unfortunately. 

Examples:

Filename `Blade.Runner.1982.Final.Cut.1080p.Blu-Ray.x264-VIAHD.mkv`
Title in mkv:  `Blade Runner (1982)`

Filename: `Blade.Runner.2049.2017.1080p.BluRay.x264-SPARKS.mkv`
Title in mkv: `Blade Runner 2049 (2017)`

Both are in the same folder "Blade Runner"

Both are detected as `Blade Runner` (the one from 1982). What can I do so Blade Runner 2049 gets recognized correctly? Preferably without manually changing the metadata in Jellyfin. I have the same issue with Flatliners.

**Comment by tibequadorian** [comment]
I had the same problem, just put `[tmdbid-335984]` at the end of the filename as specified in https://jellyfin.org/docs/general/server/media/movies/
